src/components/cards/LoginCard.vue

```diff
@@ -1,7 +1,19 @@
 <script setup lang="ts">
+  import { parseISO, isWithinInterval } from 'date-fns';
+  import { computed, type ComputedRef } from 'vue';
   import { type RouteLocationNormalizedLoaded, useRoute } from 'vue-router';
 
   const route: RouteLocationNormalizedLoaded = useRoute();
+
+  // Define the maintenance date range
+  const isMaintenancePeriod: ComputedRef<boolean> = computed(() => {
+    const today: Date = new Date();
+    const startDate: Date = parseISO('2024-12-07');
+    const endDate: Date = parseISO('2024-12-08');
```

Comment:
Ergibt auf meinem System:
2024-12-06T23:00:00.000Z
2024-12-07T23:00:00.000Z

Ich weiß nicht, welche Uhrzeit der Server hat. Bitte mit Christian absprechen, ob das ok ist. Ansonsten mit Zeitangabe korrigieren.

Reply:
Dein System zeigt korrekterweise die Dates in UTC aber die App sollte das korrekt behandeln: Für uns ist 
2024-12-06T23:00:00.000Z dann 2024-12-07 00H weil Deutschland bei UTC+1 liegt z.B.

Reply:
Ich habe trotzdem jetzt die Zeit hinzugefügt aber sollte nichts ändern.